stdlib/public/Concurrency/Task+TaskExecutor.swift

```diff
@@ -128,12 +128,12 @@ import Swift
 /// - Throws: if the operation closure throws
 /// - SeeAlso: ``TaskExecutor``
 @_unavailableInEmbedded
-@available(SwiftStdlib 9999, *)
+@available(SwiftStdlib 6.0, *)
 @_unsafeInheritExecutor // calling withTaskExecutor MUST NOT perform the "usual" hop to global
-public func withTaskExecutorPreference<T: Sendable>(
+public func withTaskExecutorPreference<T: Sendable, E>(
```

Comment:
T doesn't need to be Sendable here

Reply:
Whoops thanks :)

Reply:
Heh turns out there's all kinds of issues this causes... I don't think this change can land yet, will discuss with the team tomorrow